CommonCPPCTest.cmake

```diff
@@ -121,9 +121,14 @@ foreach(FILE ${TEST_FILES})
   endif()
 endforeach()
 
+set(__CONSOLE)
+if(CMAKE_VERSION VERSION_GREATER 3.2.2)
```

Comment:
Seems like 3.2 is enough: http://www.cmake.org/pipermail/cmake/2015-March/060082.html